rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+---
+feature: community-team
+start-date: 2021-07-31
+author: Irenes (Irene Knapp)
+co-authors: ashkitten
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Establish an official community team to make moderation decisions, model social norms, and mediate interpersonal problems.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not sustainable to have to call on Graham any time there's a spammer or conflict that requires moderation, and we'd like to help the community become more self-regulating.
+
+# Goals
+[goals]: #goals
+
+* Model and enforce social norms
+* Mediate disputes when doing so is beneficial to the community
+* Respect the privacy of community members
+* Foster newcomers and develop their leadership potential
+* Moderate all places where community members interact, including Discourse, Matrix, and GitHub
+* Ensure that collaboration on shared technical goals is possible
+* Ensure that the Nix community is a welcoming space for people from marginalized backgrounds
+* Do not allow the Nix community to be a place for spreading ideas rooted in fascism or bigotry
+* Ensure that people in positions of authority within the community, whether technical or otherwise, promote community participation in all things and use their authority in ways that are consistent with the community's values
+* Don't be overly driven by the personalities of a small set of people
+* Maintain a good ratio of experienced members of the community team to newcomers, such that newcomers can gain experience as they work
+* Avoid excessive churn on the community team
+
+# Non-Goals
+[non-goals]: #non-goals
+
+* Provide technical leadership
+* Create a hierarchical power structure
+
+# Detailed design
+[design]: #detailed-design
+
+The community team is not an elected position, it is a collection of people who are interested in helping the community thrive. Membership is not permanent, and all community members are encouraged to participate at some point regardless of previous community moderation experience. It isn't important whether or not members are the best suited for the position, we want to avoid making it an exclusive position or a position associated with only certain people. We also want to avoid having people on the community team get overly attached to their personal role.
```

Comment:
The vast majority of lines in the accepted RFCs wrap at less than 80 characters making for far more accessible reading. 

Why is this document so different?

Reply:
The simple answer is that the RFC template didn't say anything about it so I went with my usual practice.

The longer answer is actually about accessibility, so thank you for mentioning it. Many text editors can do soft line wraps, which allows the reader to customize the width to what they prefer. However, if a document has hard line wraps, re-wrapping features can't work as smoothly - they have to guess where the paragraph breaks are supposed to be. In the absence of guidance to the contrary, I consider it an accessibility best-practice to format documents in ways that allow the reader to tailor the experience to their liking.

This is a decision I made many years ago after contemplating the trade-offs; it wasn't a choice I made specifically for the document. I regret that you find it hard to read.

Reply:
On the object level I agree it is unfortunate that our remplate is line wrapped. On the meta level this is of course a part of the signal of not having been «annoyed by heart» by the minor unfortunate details of the decision making in this project…

Reply:
> This is a decision I made many years ago after contemplating the trade-offs; it wasn't a choice I made specifically for the document.

This along with your aggressive and verbose responses is another signal here of your overbearing and condescending attitude, and I find it insulting.

Many years ago *I* did a small literature review of ideal line length and found that reading comprehension greatly increases with narrower line width (within the range of acceptable widths, from newspaper articles to lines that span hundreds of characters - like yours).

Working in technology organizations where text is committed and edited by many authors, I have come to appreciate tight line wraps a great deal. In addition to better comprehension, it makes for much easier side-by-side diffing. Having to scroll to the right to read a diff is rude to the reader.

My impression is that by putting your text in long lines like this, and by writing *so* verbosely, you are communicating that you don't want us to read, understand, or contribute to this document. By limiting this access, you are trying to set yourself up as a community dictator - you will have a much smaller number of people to contend with if you engage in disputes. I think most would rather leave than deal with you.

Let me make a suggestion. Start over. Make an empty outline and a statement of purpose. Stop aggressively responding to every critic and let others do some thinking for themselves. Be more of an organizer of *everyone's* ideas than a coder who aggressively writes, reviews, and pushes their own code into production.

Reply:
> Many years ago I did a small literature review of ideal line length and found that reading comprehension greatly increases with narrower line width (within the range of acceptable widths, from newspaper articles to lines that span hundreds of characters - like yours).

I really hope nobody reads those unwrapped, given that both browsers and text editors support automatic wrapping. And from the point of view of short lines, the worst case is exactly wrapping 80-character lines to slightly shorter length because of font size preferences and screen size.

Reply:
> > This is a decision I made many years ago after contemplating the trade-offs; it wasn't a choice I made specifically for the document.
> 
> This along with your aggressive and verbose responses is another signal here of your overbearing and condescending attitude, and I find it insulting.
> 

I'm doing all I can to be as polite as possible, even in the face of attacks on my character. I'm doing that because writing this RFC is, to me, a form of public service, and sometimes public service means listening to criticism even when that criticism is unfairly personal. As much as I would like to simply ignore this particular complaint, because it is personal, I feel that I must respond to it by saying that I don't see how anything I've said is in any way aggressive.

On the contrary, I have to act on the assumption that you're simply applying the stereotype that any female-presenting person who stands up for themselves is aggressive because we aren't "supposed" to have opinions. I have to assume that because you're not giving me anything else to work with; your explanation for this "aggression" is simply that I'm responding to criticism with careful thought rather than with blanket statements.

I am taking the time to address criticism because I believe in the proposal and am trying to help people work through their concerns. Of course if anyone tells me they don't want to hear responses from the RFC authors, that's fine and I'm happy to let them talk on their own, but I don't understand how the RFC discussion period can proceed without *discussion*. I don't see how simply responding to people's questions can be aggression. The responses are long because the issues are important and nuanced and everyone here, including me, is trying to talk through complex topics which can't be boiled down to a simple yes or no.

With all that said, of course, it's not for me to decide how you feel about my words, @aaronchall. It sounds like you are asking me to stop responding to you. You don't have a right to ask that on other people's behalf, but you can certainly ask it for yourself. I will honor that request, although it may be awkward in later steps of the RFC process where RFC authors are obligated to respond to questions. If you want me to respond to you in future, please be very explicit about asking for it.